packages/cli/src/index.ts

```diff
@@ -0,0 +1,3 @@
+import { cli } from './lib/cli';
+
+cli(process.argv[process.argv.length - 1]);
```

Comment:
main, and index are 3 different this things. is index being used as a executable ? 

Because it probably should only import and export things.

Reply:
This will be restructured later by @BioPhoton in #6, I only included the bare minimum here for testing the config imports work end-to-end.